pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,31 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "kubernetes.io/gce.loadbalancer.allow.source.range"
 )
 
+//Validation Functions for expecting annotations in GCE
+var gceAnnotationValidation = map[string]cloudprovider.ValidationFunc{
```

Comment:
I was thinking if there are annotations added in the future. Maybe we can validate them in one shot. But seems over complicates stuff here. So I will keep it simple